### PR TITLE
Tika api integration

### PR DESCRIPTION
**Pull request by joecks**
## What 🕵️ 🔍

- What does the pull request add and/or solve?

## References 📝 🔗

- [Jira task](https://xainag.atlassian.net/browse/)

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.

### CODE REVIEW

File: b2b_personalization/infra/dev/eu-central-1/b2b/tenants/xayn/documents_api/terragrunt.hcl [hunk 1]
```diff
@@ -15,7 +15,28 @@ include "env" {
...
+  mock_outputs = {
+    dns_name = "http://localhost"
+  }
+  mock_outputs_allowed_terraform_commands = ["validate", "providers"]
+}
+
+
 inputs = {
+  tika_configuration = {
+    allowed_conentent_type = []
```
Comment (acrrd) [inline b2b_personalization/infra/dev/eu-central-1/b2b/tenants/xayn/documents_api/terragrunt.hcl:35]: This has been renamed `allowed_media_type`

File: b2b_personalization/infra/dev/eu-central-1/b2b/tenants/xayn/documents_api/terragrunt.hcl [hunk 2]
```diff
@@ -15,7 +15,28 @@ include "env" {
...
+    enabled                = true
```
Comment (acrrd) [inline b2b_personalization/infra/dev/eu-central-1/b2b/tenants/xayn/documents_api/terragrunt.hcl:36]: We must also add 
`extractor = "tika"`

**Review by acrrd — COMMENTED** [review]
The PR is still under review so something it could change. My comments refer to the current status.

File: b2b_personalization/infra/dev/eu-central-1/b2b/tika_api/terragrunt.hcl [hunk 3]
```diff
@@ -0,0 +1,7 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+include "env" {
+  path = "${get_terragrunt_dir()}/../../../../_env/tika.hcl"
+}
```
Comment (Robert-Steiner) [inline b2b_personalization/infra/dev/eu-central-1/b2b/tika_api/terragrunt.hcl:7]:
```suggestion
}

```

**Review by Robert-Steiner — APPROVED** [review]

**Pull request merged**